rllib/policy/eager_tf_policy_v2.py

```diff
@@ -557,11 +561,29 @@ def compute_log_likelihoods(
             dist_inputs, self.dist_class, _ = self.action_distribution_fn(
                 self, self.model, input_batch, explore=False, is_training=False
             )
+            action_dist = self.dist_class(dist_inputs, self.model)
         # Default log-likelihood calculation.
         else:
-            dist_inputs, _ = self.model(input_batch, state_batches, seq_lens)
+            if self.config.get("_enable_rl_module_api", False):
+                if in_eval_mode:
+                    self.model.eval()
+                    output = self.model.forward_exploration(input_batch)
+                else:
+                    output = self.model.forward_train(input_batch)
 
-        action_dist = self.dist_class(dist_inputs, self.model)
+                action_dist = output.get(SampleBatch.ACTION_DIST)
+
+                if action_dist is None:
```

Comment:
move this check to be after the big if-else, so all these cases get validated?

Reply:
The message in the check only applies to the RLModule API stack. not so much to the old ModelV2 stack. Only in the new stack it can be empty. because in the other cases we are constructing them in their own conditions.

Reply:
ok